cmd/apply/cmdapply.go

```diff
@@ -147,20 +151,24 @@ func (r *ApplyRunner) RunE(cmd *cobra.Command, args []string) error {
 		r.printStatusEvents = true
 	}
 
-	ch := a.Run(ctx, inv, objs, apply.Options{
-		ServerSideOptions: r.serverSideOptions,
-		PollInterval:      r.period,
-		ReconcileTimeout:  r.reconcileTimeout,
-		// If we are not waiting for status, tell the applier to not
-		// emit the events.
-		EmitStatusEvents:       r.printStatusEvents,
-		NoPrune:                r.noPrune,
-		DryRunStrategy:         common.DryRunNone,
-		PrunePropagationPolicy: prunePropPolicy,
-		PruneTimeout:           r.pruneTimeout,
-		InventoryPolicy:        inventoryPolicy,
-	})
-
+	ch := make(chan event.Event)
```

Comment:
Once printers are refactored, this goroutine and channel will go away. A printer (0 or more) will be passed to Run as a callback. Not done here to reduce scope.